### PR TITLE
Refactored non-null returning components & their tests

### PR DESCRIPTION
**Pull request by DemonusPC**
The Headline and SubHeading now return a full null when its props are null. The null test for these components have been refactored. These are the last two components that need to be changed. 

* [x] Fixes https://github.com/bbc/simorgh/issues/209
* [x] Tests added for new features

* [ ] Test engineer approval

### CODE REVIEW

File: src/app/components/Headline/index.jsx [hunk 1]
```diff
@@ -10,7 +11,11 @@ const StyledHeadline = styled.h1`
 `;
 
 const Headline = ({ blocks }) => {
-  const { text } = blocks[0].model.blocks[0].model;
+  const { text } = extractText(blocks);
+
+  if(!text){
```
Comment (jtart) [inline src/app/components/Headline/index.jsx]: Surprised linting isn't complaining about this. A trivial change: add spacing to this conditional, e.g. `if (!text) {`

File: src/app/components/SubHeading/index.jsx [hunk 2]
```diff
@@ -1,8 +1,13 @@
 import React from 'react';
+import { extractText } from '../../helpers/blocks';
 import { textPropTypes, textDefaultPropTypes } from '../../helpers/proptypes';
 
 const SubHeading = ({ blocks }) => {
-  const { text } = blocks[0].model.blocks[0].model;
+  const { text } = extractText(blocks);
+
+  if(!text){
```
Comment (jtart) [inline src/app/components/SubHeading/index.jsx]: Surprised linting isn't complaining about this. A trivial change: add spacing to this conditional, e.g. `if (!text) {`

File: src/app/components/Headline/index.test.jsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import Headline from './index';
 import { textBlock } from '../../models/blocks';
-import {shouldMatchSnapshot} from '../../helpers/tests/testHelpers';
+import {shouldMatchSnapshot, isNull} from '../../helpers/tests/testHelpers';
```
Comment (jtart) [inline src/app/components/Headline/index.test.jsx]: Surprised linting isn't complaining about this. A trivial change: add spacing to this destructure, e.g. `{ shouldMatchSnapshot, isNull }`

**Review by jtart — CHANGES_REQUESTED** [review]

**Review by jtart — APPROVED** [review]

**Review by jp-bbc — APPROVED** [review]
Fine. Approved.

**Pull request merged**